renovate-changesets/dependencyConfig.ts

```diff
@@ -0,0 +1,10 @@
+import * as core from '@actions/core';
+
+export const getDependencyManager = (): string => {
+  return core
+    .getInput('dependency-manager', {
+      required: false,
```

Comment:
It's set with a default in the yaml file. But still marked as not required here? What's even the return type here, can `.trim` below get an undefined error if the default wasn't there? See the comment above; maybe it's better to just make two actions that are specialized, or change around how this works a bit